docs/howtos/converting-a-component-to-uniffi.md

```diff
@@ -7,8 +7,7 @@ hand-written FFI layer and foreign-language bindings.
 As we've gained more experience with building components in this way, we've
```

Comment:
Let's split the doc changes into a separate PR (but thanks for doing them!)

Another section we discovered is missing is "converting your errors" if you'd like to add even more :)